### PR TITLE
Better npm scripts

### LINKED ISSUE #78: NODE_ENV prefix throws error in Windows
```
> react-toolbox@0.11.5 build E:\GitHub\react-toolbox
> NODE_ENV=production npm run babel && npm run sass

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.

npm ERR! Windows_NT 10.0.10240
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "build"
npm ERR! node v4.2.2
npm ERR! npm  v3.3.12
npm ERR! code ELIFECYCLE
npm ERR! react-toolbox@0.11.5 build: `NODE_ENV=production npm run babel && npm run sass`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-toolbox@0.11.5 build script 'NODE_ENV=production npm run babel && npm run sass'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the react-toolbox package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     NODE_ENV=production npm run babel && npm run sass
npm ERR! You can get their info via:
npm ERR!     npm owner ls react-toolbox
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     E:\GitHub\react-toolbox\npm-debug.log
```

Seems prefixing the environment variable when invoking an NPM script doesn't work in Windows when checking out the project source and running `npm install` (**Note:** Getting from NPM as a dependency still works fine)

### PR DESCRIPTION
**Pull request by NogsMPLS**
using better-npm-run for cross platform node_env and uv_threadpool_size.

Fixes #78

### CODE REVIEW

**Comment by soyjavi** [comment]
Hi @NogsMPLS we can not (auto) merge your PR.. could you update it?

best

**Comment by soyjavi** [comment]
Hi @NogsMPLS we can not (auto) merge your PR.. could you update it?

best

**Comment by javivelasco** [comment]
I coudn't `npm install` and run in OSX :disappointed: 
Can anybody try? I'd like to make life easier to those running on Windows.
Thanks for the PR @NogsMPLS!

**Comment by javivelasco** [comment]
@NogsMPLS I think the tests were not passing because `sinon` dependency was pointing to a repo with version 2 that is no longer working. I already fixed it in master by making the dependency point to the official branch of version 2 in the Sinon repository.

Please, merge master against your branch and see if tests are passing. Anyway, I couldn't make it work in my Macbook :disappointed:

**Comment by NogsMPLS** [comment]
ya, i think it is not passing because the ```build``` script is running before thennew package gets installed.

since build now starts with ```better-npm-run``` command, and trabis executrs it before better-npm-run is installed, it doesnt know what to do.

**Comment by javivelasco** [comment]
Make sense. Maybe `cross-env` can work too? Check the example repo, a guy sent a PR a couple of days ago!

**Comment by NogsMPLS** [comment]
Haha, I just started looking into cross-env. I'll see if i can get it to work.

**Comment by NogsMPLS** [comment]
hrmm...im running into spawn ENOENT issues with cross-env and the latest version of better-npm-run.

I can get better-npm-run 0.0.3 working no problem on my windows machine, but it doesn't seem to want to pass travis build. :confounded:

**Comment by javivelasco** [comment]
Damn, maybe we can find a similar tool...

**Comment by NogsMPLS** [comment]
i have a possible fix in here: https://github.com/kentcdodds/cross-env/pull/3

If that PR turns out to be good, then i'll try the new version of cross-env and maybe that'll work :pray:

**Comment by javivelasco** [comment]
Nice! :)

**Comment by NogsMPLS** [comment]
yaaay, passed tests and works on my windows machine, and it's a fairly minor visual change, so readability is better than ```better-npm-run``` as well.

Next up I'm gonna try and figure out how to find a workaround for #83

**Comment by javivelasco** [comment]
Awesome! Let us know when it's ready :)
Thanks @NogsMPLS

**Comment by NogsMPLS** [comment]
this pr is ready to merge, #83 is an unrelated issue

File: docs/package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
     "babel-eslint": "^4.1.3",
     "babel-loader": "^5.3.2",
     "babel-plugin-react-transform": "^1.1.1",
+    "better-npm-run": "0.0.3",
```
Comment (javivelasco) [inline docs/package.json]: Missing dependency?

**Comment by NogsMPLS** [comment]
good catch, thought i deleted that from the commit, my bad. i'll resubmit here in a minute

**Comment by javivelasco** [comment]
Thanks!

**Pull request merged**